docs/reference/item.md

```diff
@@ -177,7 +177,43 @@ item of the reviewed attribute, not the parent item of the link).  The
 fingerprint of the link is does **not** contribute to the fingerprint of the
 item.
 
-## `ref`
+## `references` (new array behavior)
+
+An array of external references. An item may reference a number of external
+files. The external references are displayed in a published document.
+
+Doorstop will search in the project root for a file matching the specified
+reference. If multiple matching files exist, the first found will be used.
+
+A file is considered a text-file unless its file extension is listed in
```

Comment:
For what is this text-file stuff necessary?

Reply:
I don't think we need this for the `references` feature, even though I think most of the time we will link text files which are easier to check than binaries.

No strong opinion about this.

Reply:
I think in this commit, we don't look at the content?